### PR TITLE
feat: make express, koa, fastify available on Deno

### PR DESCRIPTION
**Pull request by KnorpelSenf**
As requested in https://t.me/grammyjs/65130

### CODE REVIEW

**Review by Loskir — APPROVED** [review]

File: src/convenience/frameworks.shared.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// deno-lint-ignore-file no-explicit-any
+
+export const SECRET_HEADER = "X-Telegram-Bot-Api-Secret-Token";
+
+/** express web framework */
+const express = (req: any, res: any) => ({
+    update: Promise.resolve(req.body),
+    header: req.header(SECRET_HEADER),
```
Comment (darvesh) [inline src/convenience/frameworks.shared.ts:8]: Shouldn't it be [request.get](https://expressjs.com/en/5x/api.html#req.get)?

Comment (KnorpelSenf) [reply src/convenience/frameworks.shared.ts:8]: It's an alias, isn't it? I think `req.header` is a more accurate term for a request header than `req.get`.

Comment (darvesh) [reply src/convenience/frameworks.shared.ts:8]: It is not anywhere in the documentation, though. Node http which Express extends, the method is `getHeaders` and `headers` is an object.

Comment (darvesh) [reply src/convenience/frameworks.shared.ts:8]: Oh sorry, I didn't read the last line. Nvm lol

Comment (KnorpelSenf) [reply src/convenience/frameworks.shared.ts:8]: It's stated at exactly the point in the docs that you linked to.
Screenshot_20220731-201047
↑ here

Comment (darvesh) [reply src/convenience/frameworks.shared.ts:8]: Yeah, sorry for the trouble :)

Comment (KnorpelSenf) [reply src/convenience/frameworks.shared.ts:8]: No worries, thanks for reviewing! :)

**Pull request merged**